docs/thermistors.md

```diff
@@ -0,0 +1,35 @@
+# Thermistors
```

Comment:
You can add this file to "docs/_data/index.yaml" so it will show up in the navigation bar.

Reply:
Fixed